src/server/game/Server/WorldSession.cpp

```diff
@@ -332,8 +326,7 @@ bool WorldSession::Update(uint32 diff, PacketFilter& updater)
                         requeuePackets.push_back(packet);
                         deletePacket = false;
 
-                        LOG_DEBUG("network", "Re-enqueueing packet with opcode {} with with status STATUS_LOGGEDIN. "
-                                    "Player {} is currently not in world yet.", GetOpcodeNameForLogging(static_cast<OpcodeClient>(packet->GetOpcode())), GetPlayerInfo());
+                        LOG_DEBUG("network", "Delaying processing of message with status STATUS_LOGGEDIN: No players in the world for account id {}", GetAccountId());
```

Comment:
why are we removing the opcode name here

Reply:
It is already logged 17 lines above. First it logs the message ID, then it logs this line, saying the processing is delayed. Makes sense?

Reply:
aight